Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -334,14 +337,298 @@ def test_fetch_incidents_with_incident_name(mocker):
     mocker.patch('ServiceNowv2.parse_date_range', return_value=("2019-02-23 08:14:21", 'never mind'))
     mocker.patch('ServiceNowv2.parse_dict_ticket_fields', return_value=RESPONSE_FETCH['result'])
     client = Client('server_url', 'sc_server_url', 'cr_server_url', 'username', 'password',
-                    'verify', 'fetch_time', 'sysparm_query', sysparm_limit=10,
+                    'verify', '2 days', 'sysparm_query', sysparm_limit=10,
                     timestamp_field='opened_at', ticket_type='incident',
                     get_attachments=False, incident_name='description')
     mocker.patch.object(client, 'send_request', return_value=RESPONSE_FETCH)
     incidents = fetch_incidents(client)
     assert incidents[0].get('name') == 'ServiceNow Incident Unable to access Oregon mail server. Is it down?'
 
 
+class TestFetchIncidentsWithLookBack:
+    LAST_RUN = {}
+
+    API_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+    NOW = datetime.utcnow()
```

Comment:
better not to use real time, such tests tend to be flaky. consider using freezegun

Reply:
done